config/routes.rb

```diff
@@ -23,6 +23,8 @@
     root to: 'projects#index'
   end
 
+  post '/test/reseed', to: 'test_utilities#reseed'
```

Comment:
Is the Heroku app's Rails environment set to anything other than `production` (eg. `test`)? If so, we could wrap this like so:

```
if Rails.env.test?
  post '/test/reseed', to: 'test_utilities#reseed'
end
```